java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -1534,6 +1534,34 @@ void testMatchesRe() {
     });
   }
 
+  @Test
+  void testContainsRe() {
+    String patternString1 = "\\d+";
+    String patternString2 = "[A-Za-z]+\\s@[A-Za-z]+";
+    String patternString3 = ".*";
+    String patternString4 = "";
+    try (ColumnVector testStrings = ColumnVector.fromStrings("", null, "abCD", "ovér the",
+        "lazy @dog", "1234", "00:0:00", "abc1234abc", "there @are 2 lazy @dogs");
+         ColumnVector res1 = testStrings.containsRe(patternString1);
+         ColumnVector res2 = testStrings.containsRe(patternString2);
+         ColumnVector res3 = testStrings.containsRe(patternString3);
+         ColumnVector expected1 = ColumnVector.fromBoxedBooleans(false, null, false, false, false,
+             true, true, true, true);
+         ColumnVector expected2 = ColumnVector.fromBoxedBooleans(false, null, false, false, true,
+             false, false, false, true);
+         ColumnVector expected3 = ColumnVector.fromBoxedBooleans(false, null, true, true, true,
```

Comment:
Note row 0 is `false` which should have been `true` for `.*`. I am writing up a cpp test to be sure and have it as a cudf issue.

Reply:
Opened https://github.com/rapidsai/cudf/issues/4671

Reply:
So should we wait for #4671 to be fixed? or would you rather move this to a separate disabled test.  Because we don't want to verify that containRe is broken.  We want a test that verifies that it does the right thing.

Reply:
Yes. That sounds ok to me. I added it anyway to that we catch it but having a breaking test does not sound like fun. I will add it as an ignored test.